packages/sanity/src/core/releases/components/ReleaseDocumentPreview.tsx

```diff
@@ -41,9 +47,33 @@ export function ReleaseDocumentPreview({
     [documentId, documentTypeName, releaseSlug],
   )
 
+  const previewPresence = useMemo(
+    () => documentPresence?.length > 0 && <DocumentPreviewPresence presence={documentPresence} />,
+    [documentPresence],
+  )
+
   return (
-    <Card as={LinkComponent} radius={2} data-as="a">
-      <SanityDefaultPreview {...previewValues} isPlaceholder={isLoading} />
+    <Card
+      tone={hasValidationError ? 'critical' : 'default'}
+      as={LinkComponent}
+      radius={2}
+      data-as="a"
+    >
+      <SanityDefaultPreview {...previewValues} status={previewPresence} isPlaceholder={isLoading}>
+        {hasValidationError && (
```

Comment:
Currently just passing the boolean rather than the individual validation errors. Once designs show how we'd want invalid documents to look, this can be udpated